client/src/components/editor/LinkAnetEntity.js

```diff
@@ -0,0 +1,29 @@
+import LinkTo from "components/LinkTo"
+import PropTypes from "prop-types"
+import React, { useEffect, useState } from "react"
+import { getEntityByUuid } from "utils_links"
+
+const LinkAnetEntity = ({ type, uuid, children }) => {
+  const [entity, setEntity] = useState()
+
+  useEffect(() => {
+    const response = getEntityByUuid(type, uuid)
+    if (response) {
+      response.then(data => setEntity(data))
+    }
+  }, [type, uuid])
```

Comment:
You miss here the getEntityByUuid dependency.

Reply:
Wait, I see lint-fix doesn't add it neither, I guess it's because it's just an import and not a prop or something else which changes.

Reply:
I am afraid I am not following you here. Line 4 does import this dependency (`import { getEntityByUuid } from "utils_links"`), or am I missing something?